README.md

```diff
@@ -23,7 +23,10 @@ How to retrieve a Yoti profile using the token
 6) [Handling users](#handling-users)-
 How to manage users
 
-7) [Support](#support)-
+7) [How to run the examples](#how-to-run-the-examples)-
+How to run the examples
```

Comment:
Makes sense. I would want to keep it as example and change it to examples after we have more than one. But honestly, it doesn't matter at all, as long as it's consistent.